.github/workflows/translations-check.yml

```diff
@@ -0,0 +1,49 @@
+name: Check Translation Files Sorting
+
+on:
+  pull_request:
+    paths:
+      - "app/assets/json/translations/react/*.json"
+
+jobs:
+  check-sorting:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Verify jq installation
+        run: |
+          if ! command -v jq &> /dev/null; then
+            echo "jq could not be found, installing..."
+            sudo apt-get update && sudo apt-get install -y jq
+          else
+            echo "jq is already installed: $(jq --version)"
+          fi
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16"
+
+      - name: Install dependencies
+        run: npm install --global jsonlint-cli
+
+      - name: Validate and check sorting of translation files
+        run: |
+          for file in app/assets/json/translations/react/*.json; do
+            echo "Checking $file"
+            jsonlint-cli $file
```

Comment:
question: did you check if `jsonlint-cli` sorts things the same way that the utility configured in our`Gruntfile.js` does? I also wonder if the other utilities in our Gruntfile actually get used, like `grunt-json-remove-fields` and `i18next-scanner`